octodns_googlecloud/__init__.py

```diff
@@ -94,9 +99,9 @@ def _apply(self, plan):
             self.log.debug("Waiting for changes to complete")
             time.sleep(self.CHANGE_LOOP_WAIT)
 
-        if gcloud_changes.status != 'done':
-            timeout = i * self.CHANGE_LOOP_WAIT
-            raise RuntimeError(f"Timeout reached after {timeout} seconds")
+            if gcloud_changes.status != 'done':
+                timeout = i * self.CHANGE_LOOP_WAIT
+                raise RuntimeError(f"Timeout reached after {timeout} seconds")
```

Comment:
If i'm understanding this change correctly it seems like it will throw the Timeout reached error the first immediately after sleeping the first time and not after giving up (120 trips) as was previously the case.

Maybe judging by this shift and the ones up above with the `_rrset_func`s maybe these indentions are accidental?